language/stdlib/modules/LibraAccount.move

```diff
@@ -79,10 +80,18 @@ module LibraAccount {
         limits_cap: AccountLimitMutationCapability,
     }
 
+    /// A resource for the number of WriteSets that has been committed on chain.
```

Comment:
Certainly the comment should be fixed up, but at this point, I'd say that it would be best to minimize the change and keep it as a separate resource, just from the perspective of reducing risk and churn in the code. I could be convinced otherwise if you really think it is worth doing....

Reply:
Good question about churn, and I would agree if this weren't already a breaking change...

I guess my opinion here is that since it is a breaking change already that we should combine these since combining them is no more "breaking" than moving the LibraWriteSetManager resource which is being done in this PR. This will need to be a genesis reset either way.

Reply:
I defer to your expertise but please be cautious....